src/snowflake/snowpark/modin/plugin/_internal/frame.py

```diff
@@ -1134,6 +1142,8 @@ def update_snowflake_quoted_identifiers_with_expressions(
                 existing snowflake quoted identifiers to new Snowpark columns.
                 As keys of a dictionary, all snowflake column identifiers are unique here and
                 must be index columns and data columns in the original internal frame.
+            data_column_snowpark_pandas_types: The optional Snowpark pandas types for the ne
```

Comment:
I think this line got cut off, unless you specifically mean not-equal-to expressions which I don't think is the case?

Reply:
Done.